assets/js/components/adminbar/AdminBarWidgets.test.js

```diff
@@ -108,7 +108,7 @@ describe( 'AdminBarWidgets', () => {
 		expect( container ).toMatchSnapshot();
 
 		expect(
-			queryByText( /Set up Google Analytics/ )
+			queryByText( /Set up Google Analytics Test/ )
```

Comment:
Looks like one test / forced failure was left behind.

```suggestion
			queryByText( /Set up Google Analytics/ )
```